wifiphisher/common/interfaces.py

```diff
@@ -387,6 +387,24 @@ def internet_access_enable(self, value):
         else:
             raise InvalidValueError(value, bool)
 
+    def nm_unmanage(self, interface):
+        """
+        Set an interface to unmanaged.
+
+        :param interface: Name of the interface
+        :type interface: str
+
+        :return: True upon success
+        :rtype bool
+        """
+
+        out = check_output(["nmcli", "dev",  interface, "managed", "no"])
+
+        if not is_managed_by_network_manager(interface):
+            raise InterfaceManagedByNetworkManagerError(interface)
+
+        return True
```

Comment:
Why `return True` here? as far as I can see this method is just to raise an error.